openmoc/docstring.i

```diff
@@ -2821,61 +2802,40 @@ the number of threads per block
 %feature("docstring") GPUSolver::computeResidual "
 computeResidual(residualType res_type) -> double  
 
-Computes the residual between source/flux iterations.  
+Computes the residual between successive flux/source iterations.  
 
 Parameters
 ----------
 * res_type :  
-    the type of residuals to compute (SCALAR_FLUX, FISSION_SOURCE, TOTAL_SOURCE)  
+    the type of residual (FLUX, FISSION_SOURCE, TOTAL_SOURCE)  
```

Comment:
Weird...it shows up as `SCALAR_FLUX` in "GPUSolver.cpp" but not here. I'll run "doxy2swig.py" again and see if that fixes things.